traits/testing/unittest_tools.py

```diff
@@ -156,16 +156,26 @@ class _TraitsChangeCollector(HasStrictTraits):
     # simultaneously.
     _lock = Instance(threading.Lock, ())
 
+    def __init__(self, **traits):
+        value = traits.pop('trait', None)
+        if value is not None:
```

Comment:
I think `if 'trait' in traits:` is better here with a `traits['monitor_trait'] = traits.pop('trait')` below.

Less confusing if initialized with `trait=None` (which will slip through undetected as an error).